Source/VCGeneration/Split.cs

```diff
@@ -139,6 +140,18 @@ void ObjectInvariant()
         }
       }
 
+      public IToken AssertToken
```

Comment:
This operation does some work: it traverses the list of commands in the block. I would reflect that by making this a method instead of a property: `GetAssertToken()`